pkgs/applications/networking/openfortigui/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, git
+, libsForQt5
+, fetchgit
+}:
+
+stdenv.mkDerivation rec {
+    pname = "openfortigui";
```

```suggestion
  pname = "openfortigui";
```
Please use two spaces for indentation